cfme/fixtures/v2v.py

```diff
@@ -41,6 +55,196 @@ def conversion_tags(appliance, host_creds):
     tag2 = appliance.collections.categories.instantiate(
         display_name='V2V - Transformation Method').collections.tags.instantiate(
         display_name='VDDK')
+    _tag_cleanup(host_creds, tag1, tag2)
     host_creds.add_tags(tags=(tag1, tag2))
     yield
     host_creds.remove_tags(tags=(tag1, tag2))
+
+
+def get_vm(request, appliance, second_provider, template, datastore='nfs'):
+    """Fixture to provide vm object"""
+    source_datastores_list = second_provider.data.get('datastores')
+    source_datastore = [d.name for d in source_datastores_list if d.type == datastore][0]
+    collection = second_provider.appliance.provider_based_collection(second_provider)
+    vm_name = random_vm_name('v2v-auto')
+    vm_obj = collection.instantiate(vm_name,
+                                    second_provider,
+                                    template_name=template(second_provider)['name'])
+
+    request.addfinalizer(lambda: vm_obj.cleanup_on_provider())
+    vm_obj.create_on_provider(timeout=2400, find_in_cfme=True, allow_skip="default",
+                              datastore=source_datastore)
+    return vm_obj
+
+
+@pytest.fixture(scope='function')
+def form_data_single_datastore(request, second_provider, provider):
+    form_data = (
+        {
+            'general': {
+                'name': 'infra_map_{}'.format(fauxfactory.gen_alphanumeric()),
+                'description': "Single Datastore migration of VM from {ds_type1} to"
+                " {ds_type2},".format(ds_type1=request.param[0], ds_type2=request.param[1])
+            },
+            'cluster': {
+                'mappings': [_form_data_cluster_mapping(second_provider, provider)]
+            },
+            'datastore': {
+                'Cluster ({})'.format(provider.data.get('clusters')[0]): {
+                    'mappings': [_form_data_datastore_mapping(second_provider, provider,
+                        request.param[0], request.param[1])]
+                }
+            },
+            'network': {
+                'Cluster ({})'.format(provider.data.get('clusters')[0]): {
+                    'mappings': [_form_data_network_mapping(second_provider, provider,
+                        'VM Network', 'ovirtmgmt')]
```

Comment:
**REQUIRED** just right above the params are used, here its hardcoded, is that intended

Reply:
while finishing scanning the pr i noted that this hardcoding was preexisting, so its perfectly valid to sort this out as a followup